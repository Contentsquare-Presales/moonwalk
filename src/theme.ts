export default {
  primary: "#fff",
  secondary: "#00adf1",
  light: "#c6ced9",
  inactive: "#828c94",
  accent: "#333"
};
