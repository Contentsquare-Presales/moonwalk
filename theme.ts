import {
  DarkTheme,
  DefaultTheme,
} from '@react-navigation/native';

export const lightTheme = {
  ...DefaultTheme,
  colors: {
    ...DefaultTheme.colors,
    secondary: '#ddd',
    background: 'rgb(241, 256, 254)',
    placeholderText: '#666',
    secondaryText: '#666',
    accent: 'rgb(254, 202, 82)',
    accentBackground: 'rgb(45, 42, 83)',
  },
};

export const darkTheme = {
  ...DarkTheme,
  colors: {
    ...DarkTheme.colors,
    text: '#eeeeee',
    background: '#2d2d2d',
    placeholderText: '#666',
    secondaryText: '#999',
    secondary: '#333',
    accent: 'rgb(254, 202, 82)',
    accentBackground: 'rgb(45, 42, 83)',
  },
};