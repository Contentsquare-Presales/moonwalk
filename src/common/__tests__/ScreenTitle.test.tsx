import React, { ReactElement } from "react";
import { ScreenTitle } from "../";
import renderer from "react-test-renderer";
import { ThemeProvider } from "styled-components/native";
import theme from "../../theme";

const renderWithTheme = (component: ReactElement) =>
  renderer.create(<ThemeProvider theme={theme}>{component}</ThemeProvider>);

describe("ScreenTitle", () => {
  it("renders correctly", () => {
    const tree = renderWithTheme(<ScreenTitle title="SAMPLE_TITLE" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
