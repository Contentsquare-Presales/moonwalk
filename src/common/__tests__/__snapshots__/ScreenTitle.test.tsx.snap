// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScreenTitle renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "justifyContent": "space-between",
        "paddingBottom": 40,
        "paddingLeft": 0,
        "paddingRight": 0,
        "paddingTop": 40,
      },
    ]
  }
>
  <Text
    style={
      Array [
        Object {
          "fontSize": 30,
          "fontWeight": "500",
          "letterSpacing": 1.5,
        },
      ]
    }
  >
    SAMPLE_TITLE
  </Text>
  <View
    accessible={true}
    clickable={false}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#00adf1",
        "borderRadius": 10,
        "opacity": 1,
        "paddingBottom": 7,
        "paddingLeft": 7,
        "paddingRight": 7,
        "paddingTop": 7,
      }
    }
  >
    <Text
      allowFontScaling={false}
      style={
        Array [
          Object {
            "color": "#fff",
            "fontSize": 20,
          },
          undefined,
          Object {
            "fontFamily": "FontAwesome5Free-Solid",
            "fontStyle": "normal",
            "fontWeight": "normal",
          },
          Object {
            "fontWeight": "900",
          },
        ]
      }
    >
      ï‚­
    </Text>
  </View>
</View>
`;
